### PR TITLE
improvement(gemini_report): inherit report from base_report with atta…

### PR DESCRIPTION
**Pull request by aleksbykov**
…chment

Gemini report  now inherity format from base_report and
added html attachment

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/report_templates/results_gemini.html [hunk 1]
```diff
@@ -43,14 +22,79 @@ <h3>
         </ul>
     </div>
     {% endif %}
-{% endblock %}
+    <h3>Nemesis Details</h3>
+    <h4>Running nemesis: {{ nemesis_name }}</h4>
```
Comment (bentsi) [inline sdcm/report_templates/results_gemini.html:26]: this is part of the `results_longevity` template. Why did you copy-paste it?

Comment (bentsi) [reply sdcm/report_templates/results_gemini.html:26]: I think we can fully use the longevity template. Is there any reason to see this table in the email body and not in the attached HTML?

Comment (aleksbykov) [reply sdcm/report_templates/results_gemini.html:26]: I use it to have the same view for email body and for attached html for gemini with nemesis. Email body will have short variant, and attached html will have full report
yes i want to reached this but we need also to have the gemini stats.

**Comment by bentsi** [comment]
done

**Pull request merged**